cli/cargo-ohrs/src/cli/init.rs

```diff
@@ -0,0 +1,15 @@
+use bpaf::{construct, positional, short, Parser};
+
+pub fn cli_init() -> impl Parser<crate::Options> {
+  let name = positional::<String>("name").help("project name");
+
+  let package_name = short('p')
+    .long("package")
+    .help("Ohpm package's name. If not set,will use project's name")
+    .argument::<String>("PACKAGE_NAME")
+    .optional()
+    .catch();
```

Comment:
`.catch()` is useful if you want to handle parse failures. It might do something if you have a non-utf8 package name, but most likely you should be able just to drop it.